app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java

```diff
@@ -52,16 +63,32 @@ public Object instantiateItem(ViewGroup container, int position) {
         return layout;
     }
 
+    /**
+     * Provides a way to remove an item from container
+     * @param container Adapter view group container
+     * @param position Index of item
+     * @param obj Adapter object
+     */
     @Override
     public void destroyItem(ViewGroup container, int position, Object obj) {
         container.removeView((View) obj);
     }
 
+    /**
+     * Interface used create on clicked callback(s).
```

Comment:
I am actually not sure what this method does, so can you please remove the Javadoc? Thanks!

Reply:
Are mentioning that destroy item comment or the one for interface

Reply:
You can remove the Javadoc of "public interface Callback {", thanks! :-)